File: src/dummy.cpp
#include <dummy.h>
#include <cstdio>

// dummy.h functions
void dummy_music(int song, sol::optional<int> fadeLenth, sol::optional<int> channelMask) {
    printf("WARNING: music disabled (-m --no-music)\n");
}

void dummy_sfx(int sfx, sol::optional<int> channel, sol::optional<int> length) {
    printf("WARNING: sfx disabled (-s --no-sfx)\n");
}

File: src/dummy.rs
use mlua::{Lua, Result};

pub fn dummy_music(_: &Lua, (_song, _fade_length, _channel_mask): (i64, Option<i64>, Option<i64>)) -> Result<()> {
	println!("WARNING music disabled: -m or --no-music flag");
	Ok(())
}